pkg/retry/error_retry.go

```diff
@@ -71,15 +80,15 @@ func (r *ErrorRetry) GetRetryBackoff(err error) (time.Duration, error) {
 	// it means the last error is retry success, and the sink is running well for some time
 	if r.lastInternalError == nil ||
```

Comment:
Whether lastInternalError should be deleted?

Reply:
I prefer to keep the logic because maybe it will be useful for some future cases. Now we can just use an infinite retry duration to avoid this.